Kajabi/sage-lib
# Sage Foundations - Font Weight Fix

## Description

Fixes incorrect font weight for headings 4-6:
- Changes font weight from `400` to `600`.

## Screenshots

|  Before  |  After  |
|--------|--------|
<img width="1193" alt="Screen Shot 2022-02-11 at 3 05 24 PM" src="https://user-images.githubusercontent.com/14791307/153670047-efee1213-a3fa-4db7-a562-5e6e14e244b0.png">|<img width="1191" alt="Screen Shot 2022-02-11 at 3 03 53 PM" src="https://user-images.githubusercontent.com/14791307/153670054-ee228f6a-674b-4663-9644-672144320023.png">


## Testing in `sage-lib`

- Navigate to Typography page.
- Check that font weights for headings 4-6 are 600.

## Testing in `kajabi-products`

(**LOW**) Fixes incorrect font weight for headings 4-6.


## Related

N/A
